spec/item_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'pry'
+require 'rspec'
+require './lib/item'
+
+RSpec.describe Item do
+  before(:each) do
+    @item1 = Item.new({ name: 'Peach', price: '$0.75' })
+    @item2 = Item.new({ name: 'Tomato', price: '$0.50' })
+  end
+
+  it 'is and Item' do
+    expect(@item1).to be_an_instance_of(Item)
+  end
+
+  describe '#initialize' do
+    it 'has a name' do
+      expect(@item1.name).to eq('Peach')
+    end
+    it 'has a price' do
+      expect(@item1.price).to eq('$0.75')
```

Comment:
Functionality that. happens in the initialize can be consolidated to the same test, if you want to save some time and some lines.